ckanext/report/controllers/blueprints.py

```diff
@@ -16,17 +18,28 @@ def redirect_to_index():
     return toolkit.redirect_to('/report')
 
 
+def view(report_name, organization=None, refresh=False):
+    body, headers = report_view(report_name, organization, refresh)
+    if headers:
+        response = Response(body)
+        for key, value in six.iteritems(headers):
+            response.headers[key] = value
+        return response
+    else:
+        return body
+
+
 reporting.add_url_rule(
     u'/report', 'index', view_func=report_index
 )
 reporting.add_url_rule(
     u'/reports', 'reports', view_func=redirect_to_index
 )
 reporting.add_url_rule(
-    u'/report/<report_name>', 'view', view_func=report_view
+    u'/report/<report_name>', 'view', view_func=view, methods=('GET', 'POST',)
 )
 reporting.add_url_rule(
-    u'/report/<report_name>/<organization>', 'org', view_func=report_view
+    u'/report/<report_name>/<organization>', 'org', view_func=view
```

Comment:
any methods?

Reply:
It defaults to GET, which should be all that's needed on that route. The report homepage needs POST for the refresh button.

Reply:
My mistake, the org route needs POST as well.